class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * WordPress Active Plugin trigger
```

Comment:
Better grammar needed, ie. `WordPress plugin activated trigger` that shows an action being just done